src/drivers/display/st_ltdc/lv_st_ltdc.h

```diff
@@ -15,7 +15,7 @@ extern "C" {
  *********************/
 
 #include "../../../lv_conf_internal.h"
-#if LV_USE_ST_LTDC
+#if LV_USE_ST_LTDC && LV_USE_OS != LV_OS_CHIBIOS
```

Comment:
I think this is not mandatory, LTDC for chibios is something that is not intented to run in other scenarios.

But the contrary is possible :)